client/components/stage.jsx

```diff
@@ -1,12 +1,20 @@
 import React from 'react';
 
 export default class Stage extends React.Component {
+
   render() {
     return (
       <div className="d-inline-block">
         <div className="card mr-4">
           <div className="card-body">
-            <h4>{this.props.title}</h4>
+            <div className="row">
+              <h3 className="col">{this.props.title}</h3>
+              <a href={`#newTask?projectId=${this.props.projectId}`}>
+                <i onClick={this.handleNewTask} className="fas fa-plus fa-2x col text-right"></i>
```

Comment:
`this.handleNewTask` isn't really a thing anymore since you are using an anchor.